mitodl/mitxpro
# Updated course catalog to match designs and use CMS data

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #1 

#### What's this PR do?
- Updates course catalog to match designs ([draft 1](https://impactbnd.invisionapp.com/share/4TQEESTEYMZ#/screens/354198017_Course_Catalog_V3), [draft 2](https://impactbnd.invisionapp.com/share/4TQEESTEYMZ#/screens/356499490))
- Adds seed data logic to create/delete Wagtail pages for the seeded courses/programs

#### How should this be manually tested?
- Un-seed and re-seed your database 
- Check the course catalog and make sure it looks correct
- Add a duration (via Wagtail page) or price (I'll include a script below) to various courses/programs and verify the course catalog output afterward

#### Any background context you want to provide?
- The UI of reflects a mix between the first draft of the designs and the second, which we got just a couple days ago
- As indicated in `collapsible.scss`, I adapted [this example](https://alligator.io/css/collapsible/) for the collapsible section of the catalog card. This isn't accessible without some JS, so I'll open a separate issue to address that

#### Screenshots (if appropriate)
![ss 2019-04-12 at 14 41 14 ](https://user-images.githubusercontent.com/14932219/56059665-8bfef280-5d32-11e9-82c9-f681f04a7ea0.png)

![ss 2019-04-12 at 14 40 49 ](https://user-images.githubusercontent.com/14932219/56059672-8f927980-5d32-11e9-9303-05c01830e553.png)

![ss 2019-04-12 at 14 41 02 ](https://user-images.githubusercontent.com/14932219/56059673-8f927980-5d32-11e9-8602-8a2beb68254e.png)

![ss 2019-04-12 at 14 41 23 ](https://user-images.githubusercontent.com/14932219/56059674-8f927980-5d32-11e9-8071-b3ca650b8c80.png)

![ss 2019-04-12 at 14 41 40 ](https://user-images.githubusercontent.com/14932219/56059675-8f927980-5d32-11e9-835f-451b60aa4f7b.png)
